src/Raven.Server/RavenServerStartup.cs

```diff
@@ -320,7 +320,8 @@ private static void MaybeSetExceptionStatusCode(HttpContext httpContext, Excepti
                 exception is DatabaseLoadTimeoutException ||
                 exception is DatabaseConcurrentLoadTimeoutException ||
                 exception is NodeIsPassiveException ||
-                exception is ClientVersionMismatchException)
+                exception is ClientVersionMismatchException ||
+                exception is SchemaErrorException)
```

Comment:
should be DatabaseSchemaErrorException

Reply:
I think this is an issue about references. Voron doesn't reference Raven.Client

Reply:
This is in Raven.Server so it shouldn't be a problem?